### PR TITLE
FOSSA scan enabled

### PR DESCRIPTION
**Pull request by idvoretskyi**
Ref.: https://github.com/cncf/foundation/issues/109

Signed-off-by: Ihor Dvoretskyi <ihor@linux.com>

### CODE REVIEW

**Comment by idvoretskyi** [comment]
cc @stefanprodan @dholbach

**Comment by stefanprodan** [comment]
This repo is using main instead of master.

**Review by squaremo — COMMENTED** [review]
Can the API key be a secret, or is it not an issue?

File: .github/workflows/fossa.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: FOSSA
+on:
+  push:
+    branches: [master]
```
Comment (squaremo) [inline .github/workflows/fossa.yml:4]:
```suggestion
    branches: [main]
```

File: .github/workflows/fossa.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  pull_request:
+    branches: [master]
```
Comment (squaremo) [inline .github/workflows/fossa.yml:6]:
```suggestion
    branches: [main]
```

**Comment by stefanprodan** [comment]
The API key can't be a secret as this is intended to run on PRs. All CNCF projects uses it like this so I don't think it's an issue.

**Pull request merged**